docs/learn/benefits-lukso-standards.md

```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
+
+### 🔘 Easier functions for developers
+
+Both [LSP7](../standards/tokens/LSP7-Digital-Asset.md) and [LSP8](../standards/tokens/LSP8-Identifiable-Digital-Asset.md) use the same function to transfer tokens, with only one different parameter type.
+
+- LSP7 [`transfer(address,address,uint256,bool,bytes)`](../contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md#transfer) function uses a **`uint256`** to transfer an **amount**.
+- LSP8 [`transfer(address,address,bytes32,bool,bytes)`](../contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md#transfer) function uses a **`bytes32`** to transfer a [**token ID**](../standards/tokens/LSP8-Identifiable-Digital-Asset.md#format-of-tokenids).
+
+Similarly, they both use the same function `authorizeOperator(address,[uint256/bytes32],bytes)` to approve new operators for a token allowance or specific token Ids.
+
+### 🗄️ Unlimited & Dynamic Metadata
+
+<div style={{textAlign: 'center', color: 'grey'}}>
+  <img
+    src={require('/img/learn/token-nft-metadata-examples.png').default}
+    alt="`Examples of metadata for different NFTs."
+    width="1200"
+  />
+<br/>
+<i>3 x different NFT collections with their unique traits and characteristics.</i>
+<br /><br />
+</div>
+
+Like Universal Profile, digital assets based on the LSP7 and LSP8 standards can hold an **unlimited quantity of metadata** for any various type of information considered relevant to the token or NFT collection. This can include:
+
+- Icons and image.
+- The creators of the tokens / NFT collection.
+- The list of exchanges where this token can be found.
+
+LSP8 also allows to create more complex and rich NFT collections where **each NFT can have its own custom metadata**. Setting metadata specific to each NFT is done using the standardized functions [`setDataForTokenId(...)` and `setDataBatchForTokenIds(...)`](../contracts/overview/NFT/set-nft-metadata.md). This is useful for instance for:
+
+- NFTs that need to hold specific items like digital clothing items for avatars, or weapons / body armors in a video game.
+- dynamic NFTs, where any information, traits or attributes of a NFT could be programmed to change or evolve overtime according to certain conditions, logics or triggers.
+
+### 📢 Notify on transfers
+
+On each token transfers, bother sender & recipient are notified that a token transfer happened. This notification includes a specific notification ID (_e.g: "I have sent / received some tokens / NFTs"_. See typeIds for more infos).
+
+In addition, it is possible to program and _"plug-in"_ custom logic contracts called [**Universal Receiver Delegate**](../standards/accounts/lsp1-universal-receiver-delegate.md) to the sender and recipient, so that they can automatically react based on these different type of notifications. For example:
```

Comment:
```suggestion
In addition, it is possible to program and _"plug-in"_ custom logic contracts called [**Universal Receiver Delegate**](../standards/accounts/lsp1-universal-receiver-delegate.md) to the sender and recipient, so that they can automatically react based on these different types of notifications. For example:
```